ofl/oswald/METADATA.pb

```diff
@@ -23,221 +23,3 @@ axes {
   min_value: 200.0
   max_value: 700.0
 }
-
```

Comment:
These fallbacks were recently added. We need to readd them.

Reply:
done